packages/core/src/config/config.ts

```diff
@@ -443,6 +445,7 @@ export class Config {
     this.enableMessageBusIntegration =
       params.enableMessageBusIntegration ?? false;
     this.enableSubagents = params.enableSubagents ?? false;
+    this.tokenEfficient = params.tokenEfficient ?? true;
```

Comment:
should this be true by default? Also can we add to the documentation if we are introducing a new settings option.